src/sanitizer.js

```diff
@@ -57,13 +57,38 @@ const _transformConfig = function transformConfig(config) {
   const allowElems = config.allowElements || [];
   const allowAttrs = config.allowAttributes || [];
   const blockElements = config.blockElements || [];
+  const dropElements = config.dropElements || [];
   const dropAttrs = config.dropAttributes || [];
-  return {
-    ALLOWED_TAGS: allowElems,
-    ALLOWED_ATTR: allowAttrs,
-    FORBID_TAGS: blockElements,
-    FORBID_ATTR: dropAttrs,
-  };
+  // https://github.com/cure53/DOMPurify/issues/556
+  const isdropElementsSet =
```

Comment:
I'm not sure but maybe deduce this boolean flag easier from within the default-assignment logic above, don't you think?

Reply:
> I'm not sure but maybe deduce this boolean flag easier from within the default-assignment logic above, don't you think?

I'm probably missing something obvious here. But do you have an example of how to simplify the boolean here?

Reply:
It seems we are checking if dropElements was set twice. Once in the `dropElements = config.dropElements || []` line and then here. I'm wondering if those could be merged, to have the same semantic thing happen just once.

Reply:
Ah, I think I'm following! I'll try tackling this in https://github.com/mozilla/sanitizer-polyfill/pull/44, since that refactor will touch these lines as well (e.g. `DEFAULT_DROP_ELEMENTS` here will become a set)